crates/node-file-trace/src/lib.rs

```diff
@@ -353,6 +312,8 @@ fn process_input(dir: &Path, context: &str, input: &[String]) -> Result<Vec<Stri
 pub async fn start(
     args: Arc<Args>,
     turbo_tasks: Option<&Arc<TurboTasks<MemoryBackend>>>,
+    module_options: Option<ModuleOptionsContextVc>,
```

Comment:
This won't work. You can't create a `Vc` outside of the turbo tasks context. If `Value<ModuleOptionsContext>` won't work, try `TransientInstance<ModuleOptionsContext>`

Reply:
Cool